benefits/core/migrations/0001_initial.py

```diff
@@ -65,6 +65,8 @@ class Migration(migrations.Migration):
                 ("unverified_title", models.TextField()),
                 ("unverified_headline", models.TextField()),
                 ("unverified_blurb", models.TextField()),
+                ("enrollment_confirmed_header", models.TextField()),
+                ("enrollment_confirmed_message", models.TextField()),
```

Comment:
This seems like the wrong key/property name -- this isn't the `enrollment_confirmed` page, it is the `enrollment_start` or `enrollment_connect` or something.

_Eligibility_ was confirmed, but enrollment has just started.

Reply:
I see your point. I had it as `confirmed_benefit` originally, but anyways, I do think `enrollment_start` is clearer.